apps/web/src/pages/Pools/CreatePoolPage/CreatePoolPage.tsx

```diff
@@ -159,8 +159,8 @@ export const CreatePoolForm = ({
               <Submit
                 color="secondary"
                 text={intl.formatMessage({
-                  defaultMessage: "Create new pool",
-                  id: "TLl20s",
+                  defaultMessage: "Create new process",
```

Comment:
Could this be **Create process** and then consolidate the string in a common place to reuse and get rid of `AsxUp5` and `wP9+aN`?

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/9759/commits/1969c4f91f1723fa21ce0cbb3d73593f7eab8857